CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fix invalid tax rates for lines - #7058 by @IKarbowiak
 - Allow seeing unconfirmed orders - #7072 by @IKarbowiak
 - Raise GraphQLError when too big integer value is provided - #7076 by @IKarbowiak
+- Make quantity field on StockInput required - #7082 by @IKarbowiak
```

Comment:
Question - Shouldn't be in breaking changes section? Assuming that you have an App that does something with Stock over API.